app/controllers/notifications_controller.rb

```diff
@@ -62,6 +62,22 @@ def update
     end
   end
 
+  def delete_confirmation
+    @notification = Notification.find_by(id: params[:notification_id])
```

Comment:
Would this make sense to extract into a before action?

Reply:
Yeah, probably at this point